backend/tests/yeomanui.spec.ts

```diff
@@ -221,6 +239,13 @@ describe('yeomanui unit test', () => {
 		});
 	});
 
+	describe("getState", () => {
+		it("getState", async () => {
```

Comment:
please change the unit tests description to be more clear - expected result, flow description and may be parameters. 
very difficult to understand what test meaning and what its purpose.